lib/matint.gi

```diff
@@ -237,7 +237,7 @@ local k,g,b,ii,m1,m2,t,tt,si,n,m,i,j,r,jj,piv,d,gt,tmp,A,T,TT,kk;
 end);
 
 BindGlobal("BITLISTS_NFIM",
-  [ [ false, false, false, false, false ], [ true, false, false, false, false ],
+MakeImmutable( [ [ false, false, false, false, false ], [ true, false, false, false, false ],
     [ false, true, false, false, false ], [ true, true, false, false, false ],
```

Comment:
One really notices how some code was clearly converted before the backtick notation was introduced... I guess we should ignore that for now, but once the libs are merged, I guess it would be good to go over that and use backticks in more places.